components/datatypes/demographic/geo.schema.json

```diff
@@ -56,7 +76,22 @@
       "$ref": "http://schema.org/GeoCoordinates"
     },
     {
-      "$ref": "#/definitions/geo"
+      "$ref": "#/definitions/countrycode"
```

Comment:
I've used all lowecase for the definition names, but I was wondering if there is a standard naming format, such as using camelcase or hyphenated names?

Reply:
at this point in time we do not enforce any naming standards on definition names